File: src/commands/docs.ts
import { Command, flags } from "@oclif/command";
import { spinner } from "@secman/spinner";
import { cli } from "cli-ux";

export default class Docs extends Command {
  static description = "Open Secman Documentation in default browser.";

  static aliases = ["doc"];

  static flags = {
    help: flags.help({ char: "h" }),
  };

  async run() {
    const { flags } = this.parse(Docs);

    const opening = spinner("📚 Opening docs...").start();

    await cli.open("https://secman.dev/docs");

    opening.stop();
  }
}

File: app/docs.go
package app

import (
	"os"
	"fmt"

	"github.com/spf13/cobra"
	"github.com/scmn-dev/browser"
)

var DocsCMD = &cobra.Command{
	Use:   "docs",
	Short: "Open Secman documentation in default browser.",
	Long: "Open Secman documentation in default browser.",
	Run: func(cmd *cobra.Command, args []string) {
		err := browser.OpenURL("https://secman.dev/docs")

		if err != nil {
			fmt.Printf("could not open browser: %s\n", err)

			os.Exit(1)
		}
	},
}
